doc/source/examples/dem/bouncing-particle/bouncing-particle.rst

```diff
@@ -0,0 +1,168 @@
+
+.. code-block:: text
+
+  dem_3d bouncing_particle_XX.prm
+
+Every simulation takes less than 30 seconds to run. A folder named according to the restitution coefficient used will be generated (``/out_xx``).
+
+Post-processing
+---------------
+A Python post-processing code called ``Bouncing_particle_post_processing.py`` is provided with this example. It is used to compare the height reached by the particle at each bounces with the analytical solution of a hard sphere bouncing on a flat plane. This analytical solution consider instantaneous collision between the particle and the wall, thus the maximum height of each bounce can be express by the following expression:
+
+.. math::
+        h_{k}^{max} = (h_0 - r_p)e^{2k} + rp
+
+with :math:`k` represent the :math:`k^{th}` bounce, :math:`h_0` the starting height and :math:`r_p` the radius of the particle.
+
+Once the 6 simulations have been run, used the following line in your command line to run the post-processing code :
+
+.. code-block:: text
+
+  python3 Bouncing_particle_post_processing.py
+
+A figure will then appear comparing the expected values results with the analytical solution.
+
+Results and discussion
+-----------------------
+Animation of a bouncing particle with different restitution coefficient (:math:`K_n` = 5E6 N/m):
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/B1n776jHdoo" frameborder="0" allowfullscreen></iframe>
+
+It can be seen that the particle with a restitution coefficient (:math:`e`) of 1.0 doesn't have any change in its rebounds. The other particles show a reduction in rebound height that seems to respect the analytical solution express earlier.
+
+Now, using the post-processing code, it is possible to compare the effect of the normal spring constant of the conservation of the kinetic energy during the collision.
+
+.. list-table::
+
+    * - .. figure:: images/kn5E4.png
+           :width: 260
+           :alt: Mesh
+           :align: center
+
+           Spring constant equal to 5E4.
+
+      - .. figure:: images/kn5E5.png
+           :width: 260
+           :alt: Mesh
+           :align: center
+
+           Spring constant equal to 5E5.
+
+      - .. figure:: images/kn5E6.png
+           :width: 260
+           :alt: Mesh
+           :align: center
+
+           Spring constant equal to 5E6.
+
+
+
+As it can be seen of the previous figures, when the normal spring constant becomes less stiff, the height of each bounce is less than what its suppose to be according to the analytical solution, except when the restitution coefficient is equal to 1.0. This is due to the assumption of instantaneous contact, which becomes false for an elastic particle. Since the particle is less stiff, the contact time between the particle and the wall is longer, thus the damping term in the force calculation comes into effect over a longer period and more kinetic energy is lost.
```

Comment:
```suggestion
As it can be seen of the previous figures, when the normal spring constant is lower, the height of each bounce is less than the analytical solution, except when the restitution coefficient is equal to 1.0. This is due to the assumption of instantaneous contact, which becomes false for a very elastic particle. In this case, the particle-wall overlap becomes significant compared to the particle diameter leading to additional energy dissipation. Essentially, particles which are overly soft break the underlying assumption of the soft sphere DEM model. The same conclusions were reached in the reference MFIX benchmark.
```